eng/pipelines/runtimelab.yml

```diff
@@ -60,8 +60,8 @@ jobs:
     jobTemplate: /eng/pipelines/common/global-build-job.yml
     buildConfig: Checked
     platforms:
-    - Linux_x64
-    - Windows_NT_x64
+    # - Linux_x64
```

Comment:
Just remove the whole section if we don't want to build and test a checked runtime on FreeBSD.

Reply:
ok, I'll do that. Time may prove me wrong but I don't see value in burning more CPU cycles right now.